pkgs/by-name/xr/xrsh/package.nix

```
@@ -0,0 +1,63 @@
+{
+  stdenv,
+  fetchFromGitea,
+  makeWrapper,
+  writeShellApplication,
+  python3Packages,
+  openssl,
+  lib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xrsh";
+  version = "0.143-unstable-2025-02-21";
```

https://codeberg.org/xrsh/xrsh/commit/5d6e9f4615e57f01ba0859000d90f4f162a06653 is from today:
```suggestion
  version = "0.143-unstable-2025-02-28";
```

Please update the commit title as well:
```
xrsh: init at 0.143-unstable-2025-02-28
```